### PR TITLE
Webpack

### PR DESCRIPTION
**Pull request by chioreandan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+        <li class="nav-item">
+          <a class="nav-link" href="#">CONTACTS</a>
+        </li>
+      </ul>
+    </div>
+  </nav>
+
+
+
+  <!-- <script src="/js/jquery.min.js"></script>
```
Comment (florinionce) [inline index.html]: remove those :D  

And the extra line :D

File: index.html [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway:400,800">
+  <link rel='stylesheet' href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+  <!--  <link rel="stylesheet" href="src/css/styles.css">
+    <link rel="stylesheet" href="src/css/styles.css">
+    <link rel="stylesheet" href="src/multirange.css">-->
+</head>
+
+<body>
+  <script src="dist/bundle.js"></script>
```
Comment (florinionce) [inline index.html]: Try and move this in the footer. See if it still works. 

it's better to have the JS loaded after the DOM rendered. Better performance

File: index.html [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+
+  <!-- <script src="/js/jquery.min.js"></script>
+            <script src="/js/popper.min.js"></script>
+            <script src="/js/bootstrap.min.js"></script>
+            <script src="/js/handlebars.min.js"></script>
+            <script src="multirange.js"></script>
+            <script src="https://unpkg.com/wnumb@1.1.0"></script>
+            <script src="/js/handlebarsTest.js"></script>-->
+  <script src="/dist/bundle.js"></script>
```
Comment (florinionce) [inline index.html]: hmm...this is duplicated :D

File: src/admin.html [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+	<meta charset="utf-8" />
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Admin</title>
+	<h1>ADMIN</h1>
+	<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Raleway:400,800">
+	<link rel='stylesheet' href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+
+<body>
+<!--	<script src="dist/admin.js"></script>-->
```
Comment (florinionce) [inline src/admin.html]: Is this intended? 

Also the indentation in this file is broken

File: src/js/admin.js [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+	for (let i = 0; i < btn.length; i++) {
+		btn[i].addEventListener("click", function () {
+			let toDel = menu[i].name;
+			menu.splice(i, 1);
+			let ls = localStorage.setItem("savedMenu", JSON.stringify(menu));
+			location.reload();
+		})
+	}
+	let pictureUrl = '';
+	$('#i_file').change(function (event) {
```
Comment (florinionce) [inline src/js/admin.js]: that ID naming is not the best. Try and find something self explanatory

File: src/js/admin.js [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+		tmppath = URL.createObjectURL(event.target.files[0]);
+		$("img").fadeIn("fast").attr('src', URL.createObjectURL(event.target.files[0]));
+		pictureUrl = tmppath;
+	});
+	modalSave.addEventListener("click", function () {
+		let name = document.getElementById('nume-produs').value;
```
Comment (florinionce) [inline src/js/admin.js]: Try and use english only - for variables, naming etc.

File: src/js/index.js [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+  else{
+    k = filterByPrice(menu,s);
+  }
+  renderMenu(k);
+  console.log()
+}
+
+function renderItems() {
+  $('index.html').html();
+  let k = menu;
```
Comment (florinionce) [inline src/js/index.js]: why do you need that `k` variable? can't you just say `renderMenu(menu)`?

File: src/js/index.js [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+  dropwdown_item.addEventListener("click", function (e) {
+    e.preventDefault();
+    currentCategory=e.target.innerHTML;
+    k = filter(menu, slider2.value,e.target.innerHTML);
+    renderMenu(k);
+  })
+
+  slider2.addEventListener("click", function(){
+    if(currentCategory!=''){
```
Comment (florinionce) [inline src/js/index.js]: I think you need `currentCategory !== ''`

Also, please fix the spacing

**Review by florinionce — COMMENTED** [review]
Nice job. 

Please check my comments 👍  👍  👍

File: src/js/index.js [hunk 9]
```diff
@@ -0,0 +1,110 @@
+var itemTemplate = require("../templates/item.hbs");
```
Comment (robikovacs) [inline src/js/index.js:1]: I think you can replace all the `var` with `let`

File: src/js/index.js [hunk 10]
```diff
@@ -0,0 +1,110 @@
...
+      itemsContainer.innerHTML += html;
+    }
+  }
+}
+
+function filterByPrice(menu, price) {
+  let res = price.split(",");
+  maxPrice = parseInt(res[1]);
+  minPrice = parseInt(res[0]);
+  leftPrice.innerHTML = "&#36;" + minPrice;
```
Comment (robikovacs) [inline src/js/index.js:22]: There is a shortcut for string interpolation (aka template literals): eg: \`whatever ${minPrice}`\ -> whatever 3
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals

File: src/scss/styles.scss [hunk 11]
```diff
@@ -277,78 +296,59 @@ section .section-title {
...
+}
+
+.demo {
+	width: -moz-max-content;
+	width: max-content;
+	text-align: center;
+	transform: scale(1);
+	margin-right: 12px;
+}
+@supports (--css: variables) {
```
Comment (robikovacs) [inline src/scss/styles.scss:309]: I think this css can be written better; Also for browser specific prefixes like -webkit -moz, there is https://github.com/postcss/autoprefixer which generates it for you.

**Review by robikovacs — COMMENTED** [review]
Nice work with the refactoring. Just a few comments🥇

**Review by robikovacs — APPROVED** [review]

**Pull request merged**